frontend/components/notFound/NotFound.tsx

```diff
@@ -0,0 +1,25 @@
+import { Stack, Typography } from '@mui/material';
+import { useRouter } from 'next/router';
+import RandomKawaii from '../../components/notFound/RandomKawaii';
+import Button from '../generic/Button';
+
+interface Props {
+  message?: string;
+}
+
+function NotFound({ message }: Props) {
+  const router = useRouter();
+  return (
+    <Stack justifyContent="center" alignItems="center" spacing={2}>
+      <RandomKawaii isHappy={false} size={200} />
+
+      {message && <Typography className="mt-1 text-gray-400">{message}</Typography>}
```

Comment:
This classnames are tailwind css classes?

Reply:
Oops